styles/index.css

```
@@ -0,0 +1,111 @@
+*{
+    /* border-style: solid;
+    border-color: black;
+    border-width: 1px; */
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+body {
+    background-color: #0E5FF7;
+}
```

If you want a rule to apply everywhere it will cascade so you can apply it to the body instead of `*`:

```suggestion
body {
    background-color: #0E5FF7;
    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
}
```